WeakAuras/Transmission.lua

```diff
@@ -1897,13 +1897,21 @@ Comm:RegisterComm("WeakAuras", function(prefix, message, distribution, sender)
     if(received.m == "d") and safeSenders[sender] then
       tooltipLoading = nil;
       local data, children, icon, icons = received.d, received.c, received.i, received.a
+      local target
+      for _, installedData in pairs(WeakAurasSaved.displays) do
```

Comment:
MatchInfo tries to do this for you, did that not work?

Reply:
It appears it does. Ill remove that unecesary code.